pandas/core/arrays/timedeltas.py

```diff
@@ -111,16 +111,9 @@ def dtype(self):
     _attributes = ["freq"]
 
     @classmethod
-    def _simple_new(cls, values, freq=None, **kwargs):
-        values = np.array(values, copy=False)
-        if values.dtype == np.object_:
-            values = array_to_timedelta64(values)
-        if values.dtype != _TD_DTYPE:
-            if is_timedelta64_dtype(values):
-                # non-nano unit
-                values = values.astype(_TD_DTYPE)
-            else:
-                values = ensure_int64(values).view(_TD_DTYPE)
+    def _simple_new(cls, values, freq=None):
+        assert isinstance(values, np.ndarray), type(values)
```

Comment:
can you add doc-string to indicate possible things values can be (e.g. i8, M8[ns])

Reply:
is object even possible?

Reply:
docstring: sure

object: yes.  The point of this PR is to be more strict about what goes to _simple_new, since too much work is going on in some of them, causing some ambiguity.

As commented elsewhere, this PR does more things that it needs to, causing some confusion.  I'll separate them out.